### DISCUSSION TITLE
System Tray will not launch

### DISCUSSION THREAD

**Discussion by schildgenb** [timmo001/system-bridge#637] [Help]
### Description

I tried looking to see if this issue was already submitted but I do not see it anywhere. I am getting a node js error for EACCESS: connect eaccess when launching the system tray. I have tried uninstalling and reinstalling and it is the same error message. I have the error message attached. 
![2021-07-24 (52)](https://user-images.githubusercontent.com/31838470/127025540-58232694-9afd-416f-9899-0aa55b48d2c9.png)

### What Platform / OS are you running?

Windows

### What version are you running?

2.3.1

**Comment by timmo001** [comment]
Is system-bridge running? (system-bridge.exe will automatically launch system-bridge-tray.exe)

**Comment by schildgenb** [comment]
I see it running yes.

Thanks,

Brad Schildgen
